web/ui/www/static/js/whisper.js

```diff
@@ -196,6 +236,13 @@ function setupUpdatePage(action) {
             return;
         }
 
+        var err = isPasswordValid(request.newPassword, "", request.email);
```

Comment:
@abilioesteves since I do not have the username I did not send. Should I implement the extraction of JWT Tokens in front-end too? So that I can extract the username of the person logged.

Reply:
Sorry, I did not understand your issue. Isn't the username at `request.username`?

Reply:
@claudiosegala, please proceed with this change.